step.yml

```diff
@@ -1,7 +1,34 @@
 title: Export iOS and tvOS Xcode archive
 summary: Export iOS and tvOS IPA from an existing Xcode archive
 description: |-
-  Exports an IPA from an existing iOS and tvOS `.xcarchive` file.
+  Manages your code signing assets and exports an IPA from an existing iOS and tvOS `.xcarchive` file. You can add multiple **Export iOS and tvOS Xcode archive** Steps to your Workflows to create multiple different signed .ipa files.
```

Comment:
I would remove the first part `Manages your code signing assets and`, since currently this is an optional feature of the step and I think the first sentence should be about the step's main functionalities.

Automatic code signing could be mentioned after this section, similarly to xcode-archive:

```
Manages your code signing assets and exports an IPA from an existing iOS and tvOS `.xcarchive` file. You can add multiple **Export iOS and tvOS Xcode archive** Steps to your Workflows to create multiple different signed .ipa files.

With this Step, you can use automatic code signing in a [CI environment without having to use Xcode](https://developer.apple.com/documentation/xcode-release-notes/xcode-13-release-notes).
  In short, the Step:
  - Logs you into your Apple Developer account based on the [Apple service connection you provide on Bitrise](https://devcenter.bitrise.io/en/accounts/connecting-to-services/apple-services-connection.html).
  - Downloads any provisioning profiles needed for your project based on the **Distribution method**.
```

Reply:
good point, thanks! :)